README.md

```diff
@@ -29,6 +29,20 @@ Now it can be updated automatically every day, using [GitHub Actions Workflows](
 
 Update date: <!--GAMFC-->2024-04-18 05:58:49<!--GAMFC-END-->
 
+## 克隆项目
+
+```bash
+git clone https://github.com/372798735/github-rank-fork.git
+```
+```bash
+cd github-rank-fork
+
+npm install
+```
+run
+```bash
+npm run start
+```
```

Comment:
https://github.com/jaywcjlove/github-rank/blob/303bad15adcc7143a211f551e3511e7cf987416f/README-zh.md?plain=1#L91-L114

The content is redundant with the following.